app/src/test/java/com/rockthevote/grommet/testdata/FakeRegistrationDao.kt

```diff
@@ -6,15 +6,15 @@ import com.rockthevote.grommet.data.db.model.Registration
 class FakeRegistrationDao : RegistrationDao {
     var insertHandler: ((Array<out Registration>) -> Unit)? = null
 
-    override fun getAll(): List<Registration> {
+    override suspend fun getAll(): List<Registration> {
         TODO("Not implemented")
     }
 
-    override fun insert(vararg registration: Registration) {
-        insertHandler?.invoke(registration)
+    override suspend fun insert(vararg registrations: Registration) {
+        insertHandler?.invoke(registrations)
     }
 
-    override fun delete(registration: Registration) {
-        TODO("Not implemented")
+    override suspend fun delete(vararg registrations: Registration) {
```

Comment:
shouldn't we have an `@Delete` annotation ?

Reply:
same for other methods, just for "completeness" ?

Reply:
Hmm, I've never done that for a Fake implementation before. The annotation still exists on the interface itself. But I'll add it if that makes more sense.